src/instructlab/clickext.py

```diff
@@ -133,6 +133,10 @@ def __init__(
         super().__init__(*args, **kwargs)
         if self.show_default:
             raise ValueError("show_default must be False")
+        if self.help:
+            raise ValueError(
+                "help must not be set, it is derived from the Field description in the Config class"
+            )
```

Comment:
not quite sure I'm following this - are we exposing this error to the user here, when the cause of error might be an action taken by a developer of the code elsewhere in the codebase? 
i think this could be confusing to users

Reply:
If `help="text"` is set in `click.Option` that uses `cls=clickext.ConfigOption,`, this will fail when the command is instantiated. So it's impossible to miss it. The CI will report that error. Does that help?

Reply:
Most of the unit tests cover that naturally as well.